apm-sniffer/optional-plugins/trace-ignore-plugin/README.md

```diff
@@ -1,6 +1,19 @@
-#How to use trace ignore plugin
-Please copy the apm-trace-ignore-plugin-x.jar to `agent/plugins`
+## Support custom trace ignore
+Here is an optional plugin `apm-trace-ignore-plugin`
+
+## Introduce
+- The role of this plugin is to filter personalized services that are tracked.
+- You can set up a number of paths that need to be ignored, means the `agent` won't send the traces to `collector`.
+- The current matching rule follows `Ant Path` match style , like `/path/*`, `/path/**`, `/path/?`.
+- Copy `apm-trace-ignore-plugin-x.jar` to `agent/plugins`, restarting the `agent` can take effect.
+- [Skywalking-使用可选插件 apm-trace-ignore-plugin](https://blog.csdn.net/u013095337/article/details/80452088) have a detailed introduction.
```

Comment:
Don't put the Chinese article in English docs. Remove it unless you have or want to write an En version.

Reply:
OK~~